lib/Logger/Logger.cpp

```diff
@@ -85,17 +84,14 @@ void Logger::log(const Level level,
     }
 }
 
-void Logger::logf(const Level level,
-                     const String& file,
-                     const __FlashStringHelper* function,
-                     uint32_t line,
-                     const char* format, ...) {
+void Logger::logf(const Level level, const String& file, const __FlashStringHelper* function, uint32_t line,
+                  const char* format, ...) {
     // reimplement printf method so that we can take dynamic list of parameters as printf does
     // (we can't simply pass these on to existing printf methods it seems)
 
     va_list arg;
     va_start(arg, format);
-    char temp[64];  // allocate a temp buffer
+    char temp[64]; // allocate a temp buffer
```

Comment:
my ocd realy like the look of aligned comments.
maybe we go for a fixed spacing befor a comment?

Reply:
Phew, that's a tough one. `clang-format` aligned neighboring comment lines like this:

```cpp
 // Includes
#include "display/bitmaps.h" // user icons for display
#include "languages.h"       // for language translation
```

...but not when there is a single comment only. Adding a fixed offset might be possible but will likely break this alignment - and will look different depending on the context. Imagine a fixed 4-space offset:

```cpp
    char temp[64];        // Comment looks okay here
    char* buffer = temp;

    char tomp[64];        // Comment looks weird here because of random indent
    size_t len = vsnprintf(temp, sizeof(temp), format, arg);
```

Reply:
I think I found something in this direction, setting


```
AlignTrailingComments:
  Kind: Always
  OverEmptyLines: 2
```

Reply:
Awesome!